amqp.go

```diff
@@ -56,6 +58,7 @@ type (
 		Body() []byte
 		DeliveryTag() uint64
 		ConsumerTag() string
+		Headers() amqp.Table
```

Comment:
Best to not mix streadway/amqp types here, because this package (wabbit) contains the interfaces that enables multiple amqp implementations. The `Option` type, in this package, was created to pass options to exchange and queues and they are converted to `streadway/amqp.Table` in the client implementation (NeowayLabs/wabbit/amqp package).

I think you have two options here: 
- Reuse the `Option` type (`amqp.Table` is just a map[string]interface{} also)
- Create a new type alias to map[string]interface{} to store the headers, could call it Header or just Table also..

What do you think?

Reply:
Thanks Tiago and got the point not to mix the types.  As `wabbit.Option` and `amqp.Table` is more of the one-to-one mapping, not only the actual underlying type, but also the purpose in each packages, I'll go for `wabbit.Option`.  Please take a look at the version 2.

And also, as I mentioned in the git commit log, this is passing around the Headers info
through the message, not to support headers AMQP exchange type.

Let me know if you see any other point to improve.

Thank you for your support, Tiago!